.github/workflows/bench-turbopack.yml

```diff
@@ -2,7 +2,7 @@ name: Benchmark Turbopack
 
 on:
   push:
-    branches: [main, staging]
+    branches: [main, "bench/*"]
```

Comment:
Not sure, but instead of `on.push`, you may want `on.pull_request` + `if`. Something like:

```yml
on:
  pull_request:
    types:
      - opened
jobs:
  run_if:
    if:  startsWith(github.head_ref, 'bench/')
```

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-based-on-the-head-or-base-branch-of-a-pull-request